external/easylogging++/easylogging++.h

```diff
@@ -1954,6 +1954,12 @@ class TypedConfigurations : public base::threading::ThreadSafe {
   const Configurations* configurations(void) const {
     return m_configurations;
   }
+  Configurations* configurations(void) {
```

Comment:
Why not just make a constructor which accepts a non-const Configurations instance? Exposing these non-const pointers seems to unnecessarily complicate reasoning about object behavior ("any other object might have a ref" is quite different from "this class may steal refs from other instances").

Reply:
I'm not sure what you want. The ctor has a non const Configuration as parameter. Post the code you want to see.